ra2ce/analysis/indirect/losses.py

```diff
@@ -445,11 +430,12 @@ def _create_result(vlh: gpd.GeoDataFrame) -> gpd.GeoDataFrame:
                         performance_row[-1]["v"],
                         performance_key,
                     )
-                    if math.isnan(row_performance_change):
+                    if (math.isnan(row_performance_change) and row_connectivity == 0) or row_performance_change == 0:
                         self._calculate_production_loss_per_capita(
                             vehicle_loss_hours, vlh_row, event
                         )
-                    elif (u, v, k) == row_u_v_k:
+                    elif (not(math.isnan(row_performance_change) and math.isnan(row_connectivity)) and
```

Comment:
watch out when you start concatenating so many `if-else` statements with rather complex comparisons.